module/applications/sheets/item-archetype-sheet.mjs

```diff
@@ -9,9 +9,35 @@ export default class CrucibleArchetypeItemSheet extends CrucibleBackgroundItemSh
   static DEFAULT_OPTIONS = {
     item: {
       type: "archetype"
+    },
+    actions: {
+      removeEquipment: CrucibleArchetypeItemSheet.#onRemoveEquipment
```

Comment:
Let's add this handler to the upstream class `CrucibleActorDetailsItemSheet`. I anticipate we'll want Backgrounds to also provide equipment someday.

I think we'll only expose the data model and UX for equipment to the Archetype for now, but we can put the underlying infrastructure in place for any `CrucibleActorDetailsItemSheet` subclass to utilize.